pkg/sql/ttl/ttljob/ttljob_processor.go

```diff
@@ -200,11 +198,11 @@ func (t *ttlProcessor) work(ctx context.Context) error {
 		// Iterate over every span to feed work for the goroutine processors.
 		var alloc tree.DatumAlloc
 		for _, span := range ttlSpec.Spans {
-			startPK, err := keyToDatums(roachpb.RKey(span.Key), codec, pkTypes, &alloc)
```

Comment:
There is no reason to convert the `Key` to a `RKey`:
Outside of `fmt.Sprintf`, the `Rkey` is only used after being converted back to a `Key` (`rKey.AsRawKey()`).
Inside `fmt.Sprintf`, `RKey.String()` delegates to `Key.String()`.